pkg/common/common.go

```diff
@@ -13,6 +13,8 @@ const (
 	CDI_VERSION = "v1.0.0"
 
 	IMPORTER_DEFAULT_IMAGE = "docker.io/kubevirt/cdi-importer:" + CDI_VERSION
+	//CLONER_DEFAULT_IMAGE = "docker.io/kubevirt/cdi-cloner" + CDI_VERSION
+	CLONER_DEFAULT_IMAGE = "docker.io/zvikorn/host_assisted_cloning:latest"
```

Comment:
obviously this will change to docker.io/kubevirt.  Can we name it cloner or cdi-cloner ?

Reply:
changed to cloner

Reply:
cdi-cloner with bring it inline with current naming convention in CDI.

Reply:
Changed to cdi-cloner